Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
```

Comment:
I don't see any problems with that and applying the same CSS property for each CSS selector is far less effective than this.